<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logger</class>
 <widget class="QDialog" name="Logger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boost Logging Example</string>
  </property>
  <widget class="QLCDNumber" name="CurrentTemperature">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>SetPoint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SetPointSelector"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ramp Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="RampRateSelector"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="AtTemperature">
      <property name="text">
       <string>At Temperature</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="StartStop">
      <property name="text">
       <string>Turn Heater On</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
